edb/server/bootstrap.py

```diff
@@ -506,7 +506,7 @@ def _process_delta(delta, schema):
 
 async def _populate_misc_instance_data(cluster):
 
-    mock_auth_nonce = scram.B64(scram.generate_nonce())
+    mock_auth_nonce = scram.generate_nonce()
```

Comment:
`json.dumps()` doesn't like `bytes`, so the correct fix is to still encode this here, and decode in edgecon.pyx:490.

Reply:
It isn't encoded anymore. It returns strings in after https://github.com/edgedb/edgedb-python/pull/60 because in the spec scram nonce is a string of ASCII-printable characters rather than an array of bytes (In the current implementation it means that base64 in moved into the function)

Reply:
Ah, so it fails because we need to bump edgedb-python?  Got it.